deployer/templates/fluentd.yaml

```diff
@@ -159,6 +159,10 @@ objects:
               value: ${USE_MUX_CLIENT}
             - name: "MUX_ALLOW_EXTERNAL"
               value: ${MUX_ALLOW_EXTERNAL}
+            - name: "BUFFER_QUEUE_LIMIT"
```

Comment:
I'm not sure I fully understand the work involved with removing it but it seemed like you thought that was the appropriate thing to do?